### DISCUSSION TITLE
Remove Warning

### DISCUSSION THREAD

**Discussion by RanggaGemilang** [teamalgoritma/community#184] [Q&A] [answered]
![warning](https://user-images.githubusercontent.com/107049922/178094399-ab1eec0a-91ff-4c3d-b026-1a071a4848ae.png)

How to remove warnings like these every time we create a new .ipynb file? Thank you :)

**Comment by kevbow** [comment, marked as answer]
Hi @RanggaGemilang, 

You can use this to remove warning completely:
```
import warnings
warnings.filterwarnings('ignore')
